base/essentials.jl

```diff
@@ -106,37 +106,16 @@ julia> convert(Rational{Int64}, x)
 6004799503160661//18014398509481984
 ```
 
-If `T` is a collection type and `x` a collection, the result of `convert(T, x)` may alias
-`x`.
+If `T` is a collection type and `x` a collection, the result of
+`convert(T, x)` may alias all or part of `x`.
 ```jldoctest
-julia> x = Int[1,2,3];
+julia> x = Int[1, 2, 3];
 
 julia> y = convert(Vector{Int}, x);
 
 julia> y === x
 true
 ```
-Similarly, if `T` is a composite type and `x` a related instance, the result of
-`convert(T, x)` may alias part or all of `x`.
-```jldoctest
-julia> x = sparse(1.0I, 5, 5);
-
-julia> typeof(x)
-SparseMatrixCSC{Float64,Int64}
-
-julia> y = convert(SparseMatrixCSC{Float64,Int64}, x);
-
-julia> z = convert(SparseMatrixCSC{Float32,Int64}, y);
-
-julia> y === x
-true
-
-julia> z === x
-false
-
-julia> z.colptr === x.colptr
-true
-```
```

Comment:
Preserving an equivalently incisive illustration of this caveat might be nice if possible?

Reply:
Do you have a proposal? This example was very ugly... :slightly_smiling_face:

Reply:
Coming up with an appropriate illustration is left as an exercise to the reader 😉.

Reply:
Slack conversation recap: An equivalently incisive illustration is tricky to come up with, so scratch this comment :).